fr/community/contributor-onboarding.md

```diff
@@ -0,0 +1,132 @@
+---
+layout: layout-community
+locales: fr
+---
+
+## A propos de ce document
+
+ > "Être un committer ne veut pas dire que vous soumettez du code, ça veut dire que vous êtes engagé sur le projet."
+ > – <a href="https://community.apache.org/contributors/" target="_blank">Apache Software Foundation</a>
+
+Dans ce document vous découvrirez notre processus d'embarquement pour les nouvelles personnes qui veulent nous rejoindre pour travailler sur Hoodie.
+
+Ce document est destiné aux membres de la communauté. Il vise à être un guide pour les étapes importantes à suivre et un rappel de comment collaborer avec cette communauté.
+
+Selon notre [Code de conduite](http://hood.ie/code-of-conduct.html), notre objectif est de fournir un espace sûr pour tous dans cette communauté et nous accueillons avec joie tout le monde, y compris les développeurs et personnes qui souhaitent devenir des contributeurs hors code.
+
+Ce document est et sera toujours en chantier. Nous essayons constamment d'améliorer notre engagement et notre culture communautaire et sommes toujours heureux d'avoir des retours!
+
+###  Qu'est-ce que les gens contribuent et où?
```

Comment:
Un typographe! The only hard part was to find the proper escape code: `&#x202F;`